quantum/action_util.c

```diff
@@ -247,7 +247,13 @@ void send_keyboard_report(void) {
     keyboard_report->mods |= weak_override_mods;
 #endif
 
-    host_keyboard_send(keyboard_report);
+    static report_keyboard_t last_report;
+
+    /* Only send the report if there are changes to propagate to the host. */
+    if (memcmp(keyboard_report, &last_report, sizeof(report_keyboard_t)) != 0) {
```

Comment:
avr errors out if you don't include strings.h.

Reply:
I guess you meant `string.h` headers for the `memcmp()` and `memcpy()` calls right? Added this to the includes.

Reply:
yes. Sorry.

Reply:
No problem, was just asking.